console/console-init/ui/src/modules/endpoints/EndpointPage.tsx

```diff
@@ -59,6 +59,7 @@ export default function AddressPage() {
     if (data && data.total > 10) {
       return (
         <TablePagination
+          id="endpoint-page-pagination"
```

Comment:
The `TablePagination` component does not seem to handle ids in its definition.

Reply:
As resolved in chat :)